cmd/root.go

```diff
@@ -21,12 +26,86 @@ var (
 )
 
 var rootCmd = &cobra.Command{
-	Use:               "in-toto",
-	Short:             "Framework to secure integrity of software supply chains",
-	Long:              `A framework to secure the integrity of software supply chains https://in-toto.io/`,
-	SilenceUsage:      true,
-	SilenceErrors:     true,
-	DisableAutoGenTag: true,
+	Use:           "in-toto",
+	Short:         "Framework to secure integrity of software supply chains",
+	Long:          `A framework to secure the integrity of software supply chains https://in-toto.io/`,
+	SilenceUsage:  true,
+	SilenceErrors: true,
```

Comment:
Please set DisableAutoGenTag: true again.

Reply:
Added DisableAutoGenTag: true to remove comments.